.travis.yml

```diff
@@ -1,48 +1,45 @@
-cache: pip
+language: python
+sudo: false
+cache:
+  apt: true
+  directories:
+    - $HOME/.cache/pip
+
+python:
+  - "2.7.13"
 
 addons:
   apt:
-    sources:
-      - pov-wkhtmltopdf
     packages:
       - expect-dev  # provides unbuffer utility
       - poppler-utils # document index
       - antiword # document index
-      - python-ldap # auth_ldap
-      - wkhtmltopdf # report_webkit
-
-language: python
-sudo: false
-
-python:
-  - "2.7"
 
 env:
   global:
   - VERSION="8.0" ODOO_REPO="OCA/OCB" LINT_CHECK="0" WEBSITE_REPO="1" TESTS="0"
+  - WKHTMLTOPDF_VERSION="0.12.4"
```

Comment:
Don't we use 0.12.1?

Reply:
I used `0.12.1` version but show the following error https://travis-ci.org/OCA/OCB/jobs/377321267#L664
and the other `.travis.yml` are using 0.12.4
 - [11.0 - 0.12.4](https://github.com/OCA/OCB/blob/dcdddbda379c9b14e66f0a8fd6d09820a7607306/.travis.yml#L20)
 - [11.0 - 0.12.4](https://github.com/OCA/OCB/blob/db85fb4f9429684bcf75f7cafb291f09fbd39888/.travis.yml#L22)

Reply:
Yeah, I'm agree with you.
FYI this was disccussed from https://github.com/OCA/maintainer-quality-tools/pull/481#issuecomment-327259328

Review the `bad security issues` from @lasley comments
And issues related using old version from @Yajo comments

And the PR to support 0.12.1 version was refused.

Reply:
Well then, let's go for this...